test/e2e/petset.go

```diff
@@ -53,8 +55,33 @@ const (
 	// Should the test restart petset clusters?
 	// TODO: enable when we've productionzed bringup of pets in this e2e.
 	restartCluster = false
+	add            = "ADD"
```

Comment:
aren't there standard api events you can use for these instead of introducing a dependency?